shell/packages/sandstorm-db/db.js

```diff
@@ -214,6 +214,8 @@ Grains = new Mongo.Collection("grains");
 //                   time a session to this grain was opened.
 //   trashed: If present, the Date when this grain was moved to the trash bin. Thirty days after
 //            this date, the grain will be automatically deleted.
+//   ownerUnread: If true, this grain has had one or mone activity events since the owner last
```

Comment:
Oh wait, you do use this below, you just use the field named "ownerSeenAllActivity" with the same set of semantics as the field in ApiTokens.